### DISCUSSION TITLE
DirectStorage 1.2

### DISCUSSION THREAD

**Discussion by DrDallas1** [SmartlyDressedGames/Unturned-3.x-Community#3842] [Ideas]
Any plans for DirectStorage 1.2 implementation into Unturned? Its an NVIDIA based software that accelerates game loading time by utilizing GPU power (rather than CPU). AMD also supports DirectStorage, and its marvelous.

This game really needs the loading time benefits, especially if you have a lot of mods.

Thanks!

**Comment by andriivitiv** [comment]
Unity doesn't support this feature yet. But Nelson is working on some asset loading improvements for the next update.

**Reply by SDGNelson** [reply]
Thanks CyberAndrii, yes this is the kind of thing to be implemented in the engine rather than the game, and assets are my main focus in this next update (still in the works).
